dev/Common/Common_themeresources.xaml

```diff
@@ -13,13 +13,15 @@
             <Thickness x:Key="TextControlBorderThemeThickness">1</Thickness>
             <Thickness x:Key="TextControlBorderThemeThicknessFocused">2</Thickness>
             <Thickness x:Key="TextControlThemePadding">10,6,6,5</Thickness>
+            <Color x:Key="SystemErrorTextColor">#FFF000</Color>
```

Comment:
><Color x:Key="SystemErrorTextColor">#FFF000</Color> [](start = 12, length = 51)

I don't think that this is correct, I think it means we will no longer reference the generic.xaml resource and will instead get this copy, which for now is the same value but these could diverge in the future.  @kmahone @ranjeshj I'm not sure what the right pattern here is, something with conditional xaml? Do we need an RS5+ version of this file since the resource was added to generic.xaml in rs5?

Reply:
Using the System Error Text Color will result in a Red colour for Light theme, and Yellow colour for Dark theme.

The Icon badges use Red, Yellow, and Green consistently across themes - and these colours are likely to be used for the In App Status banners being considered #913

Reply:
@mdtauk Thank you for your insight! The colours used for this update was chosen and approved by the designer who handed me the specs. These colours are already set in the Resource Theme Dictionaries for RS5+, but I added them in Common Themeresources as there were downleveling issues for RS4 and below.

Reply:
I still think this is wrong, while this custom color you've added to common_themeresources matches the generic one now, it might not in the future. I think we need to find a way to use this in rs4 and below and use generic.xaml in rs5+

Reply:
> I still think this is wrong, while this custom color you've added to common_themeresources matches the generic one now, it might not in the future. I think we need to find a way to use this in rs4 and below and use generic.xaml in rs5+. @ranjeshj thoughts?

@kmahone

Reply:
hardcoding colors here seem incorrect. I assume we do not have any existing keys that match this color? do we need to add a new key or update an existing one ?

---
In reply to: [366998130](https://github.com/microsoft/microsoft-ui-xaml/pull/1759#discussion_r366998130) [](ancestors = 366998130)